src/rhsm/profile.py

```diff
@@ -36,8 +36,16 @@
 except ImportError:
     yum = None
 
+try:
+    import zypp_plugin
```

Comment:
This module is imported but it is not used at all. Style checker complains about it:
```
running flake8
src/rhsm/profile.py:40:5: F401 'zypp_plugin' imported but unused
```
You can solve it using something from this module or you can add following comment to the code:
```python
# noqa: F401
```

Reply:
Thanks!